includes/class-fs-plugin-updater.php

```diff
@@ -769,6 +866,52 @@ function plugins_api_filter( $data, $action = '', $args = null ) {
 
                 $data->version       = $new_version->version;
                 $data->download_link = $new_version->url;
+
+                if ( isset( $new_version->readme ) && is_object( $new_version->readme ) ) {

```

Comment:
@fajardoleo same comment as the previous, I don't see any changes to `get_latest_download_details()` so I don't understand how it will ever return the data with the readme.

Reply:
@vovafeldman, `get_latest_download_details()` calls `_fetch_latest_version()` which in turn calls `_get_latest_version_endpoint()` which is responsible for the inclusion of the `readme` param.